backend/src/routes/pathway.ts

```diff
@@ -0,0 +1,42 @@
+import { Router } from 'express';
+import Pathway from 'models/pathway';
+
+const router = Router();
+
+router.get('/', (req, res) => {
+  Pathway.find({}, (err, product) => {
+    if (err) res.send('Error getting all pathways');
+    else res.send(product);
+  });
+});
+
+router.put('/:id', (req, res) => {
+  if (req.params.id !== req.body.id) res.status(409).send('Pathway id does not match URL id');
+  Pathway.findOneAndUpdate(
+    { id: req.params.id },
+    req.body,
+    { overwrite: true, new: true, upsert: true },
+    (err, product) => {
+      if (err) res.send(err);
+      else res.status(201).send(product);
```

Comment:
Is there a way to check whether it was an update or a new document? For [PUT](https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html#sec9.6) it SHOULD return 200 on update (although since its a should its no biggie to return 201)

Reply:
Return is now 201 on initial PUT, and 200 on all subsequent calls.